.vscode/settings.json

```diff
@@ -8,5 +8,6 @@
       ],
       "url": "https://raw.githubusercontent.com/pachyderm/pachyderm/master/src/internal/jsonschema/pps_v2/CreatePipelineRequest.schema.json"
     }
-  ]
+  ],
+  "circleci.persistedProjectSelection": [],
```

Comment:
Was this intentional?

Reply:
no, and for some reason it persisted even when i thought i had removed it. removed now